manual/src/Makefile

```diff
@@ -122,9 +122,6 @@ etex-files: $(FILES)
 	$(TEXQUOTE) < $< > $*.texquote_error.tex
 	mv $*.texquote_error.tex $@
 
-version.tex: $(ROOTDIR)/VERSION
-	sed -n -e '1s/^\([0-9]*\.[0-9]*\).*$$/\\def\\ocamlversion{\1}/p' $< > $@
-
```

Comment:
`version.tex` is included in the `FILES` variable in `manual/src/Makefile` which means it gets removed by the clean target. Something needs to happen to so that `distclean` removes `version.tex` instead.